### PR TITLE
Added 2 more C repositories

### PR DESCRIPTION
**Pull request by hedyhyry**
not finished

### CODE REVIEW

**Comment by hedyhyry** [comment]
The two don’t need any labels because it’s already really beginner-friendly. 😃

**Comment by hedyhyry** [comment]
@vvscode Ok sorry but I’ll do it later

File: README.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 ## C
 
 - [Neovim](https://github.com/neovim/neovim/labels/good%20first%20issue) _(label: good first issue)_ <br> Vim-fork focused on extensibility and agility.
-- [C](https://github.com/hedyhyry/c) <br> Example, scattered C-program files.
-- [beginners-C-program-examples](https://github.com/gouravthakur39/beginners-C-program-examples) <br> Fairly short and simple beginners friendly C language programs.
+- [C](https://github.com/hedyhyry/c/labels/good%20first%20issue) _(label: good first issue)_ <br> Simple, scattered C-program examples.
+- [beginners-C-program-examples](https://github.com/gouravthakur39/beginners-C-program-examples/issues?q=label%3A%22good+first+issue%22) _(label: good first issue)_ <br> Fairly short and simple beginners-friendly C language programs
```
Comment (vvscode) [inline README.md]: please, use link directly to labeled issues, like you did above

**Comment by MunGell** [comment]
These repositories do not follow contributing guidelines.
This list is for projects that have a community of developers around them and a list of tasks that they can delegate to new contributors and help them with advice if so will be needed.

**Pull request closed without merging**